app/models/measurement.rb

```diff
@@ -24,7 +24,6 @@ class Measurement < ActiveRecord::Base
   Y_SIZES = (1..300).map { |i| 1.2 ** i * 0.000001 }
   FIXED_MEASUREMENTS_IN_A_DAY = 1440
 
-  # belongs_to :session, :through => :stream, :inverse_of => :measurements, :counter_cache => true
```

Comment:
We don't need this association anymore? Can it influence performance?

Reply:
It was already commented out. Just removed the comment ;)